### ISSUE TITLE
"Sort by rising" sorts by new instead of rising

### ISSUE THREAD

**Issue by Jolt151** [zagaberoo/diode#78]
Pics: 
diode: https://i.imgur.com/E9X13Sw.png 
browser: https://i.imgur.com/yYqMyBW.png

**Comment by zagaberoo** [comment]
Thanks for the report.

This should be a very easy fix.  Either that, or it'll be too much trouble for me to diagnose.  I'll take a look in the next week or so and we'll find out!

**Comment by Jolt151** [comment]
If you click on new, you get an option to choose between new and rising.

**Comment by zagaberoo** [comment]
Yep, you're totally right.  As you can probably tell I'm just the steward of this code rather than the author (it used to be Reddit is Fun back in the day before the source was closed).

The rising feature does seem to be bugged, so I'll put it back in the queue to get examined.
